gdrive/export_api.py

```diff
@@ -103,6 +107,11 @@ async def survey_upload_response_task(request):
                 survey_resp["skin_tone"],
             )
 
+            if upload_result:
```

Comment:
does upload_result contain anything loggable?

Reply:
Data about the spreadsheet the api call hit

Reply:
I logged the spreadsheet id, might be nice to be sure the value uploaded to the correct sheet.